packages/dart_frog_cli/test/src/commands/update/update_test.dart

```diff
@@ -59,6 +105,54 @@ void main() {
       );
     });
 
+    group('--verify-only', () {
+      test(
+        'logs latest available version',
+        () async {
+          when(
+            () => pubUpdater.getLatestVersion(any()),
+          ).thenAnswer((_) async => latestVersion);
+          when(() => argResults['verify-only']).thenReturn(true);
+
+          final result = await command.run();
+
+          expect(result, equals(ExitCode.success.code));
+          verify(
+            () => logger.info('A new version of $packageName is available.\n'),
+          ).called(1);
+          verify(
+            () => logger.info('The latest version: $latestVersion'),
+          ).called(1);
```

Comment:
In case the above suggestion is accepted:

```suggestion
        verify(
            () => logger.info(
              styleBold.wrap('The latest version: $latestVersion'),
            ),
          ).called(1);
```